jobs/package-dockertested/Jenkinsfile

```diff
@@ -21,23 +21,33 @@ node('openshift-build-1') {
 	]])
 	// https://issues.jenkins-ci.org/browse/JENKINS-33511
 	env.WORKSPACE = pwd()
+	def packages = ['docker', 'container-selinux', 'container-storage-setup', 'skopeo', 'atomic', 'python-pytoml']
+	def installed_packages = []
+
 	stage ('Check to see if we need to run') {
-	    next_docker = latestVersion('docker', 'rhel7next*')
-	    next_cselinux = latestVersion('container-selinux', 'rhel7next*')
-	    next_cstorage = latestVersion('container-storage-setup', 'rhel7next*')
-	    next_skopeo = latestVersion('skopeo', 'rhel7next*')
-	    next_atomic = latestVersion('atomic', 'rhel7next*')
-	    echo "rhel7next: docker-${next_docker} container-selinux-${next_cselinux} container-storage-setup-${next_cstorage} skopeo-${next_skopeo} atomic-${next_atomic}"
-	    test_docker = latestVersion('docker', 'dockertested')
-	    test_cselinux = latestVersion('container-selinux', 'dockertested')
-	    test_cstorage = latestVersion('container-storage-setup', 'dockertested')
-	    test_skopeo = latestVersion('skopeo', 'dockertested')
-	    test_atomic = latestVersion('atomic', 'dockertested')
-	    echo "dockertested: docker-${test_docker} container-selinux-${test_cselinux} container-storage-setup-${test_cstorage} skopeo-${test_skopeo} atomic-${test_atomic}"
-	    if ( next_docker == test_docker && next_cselinux == test_cselinux && next_cstorage == test_cstorage && next_skopeo == test_skopeo && next_atomic == test_atomic) {
+		def versions = [:]
+		for(package in packages) {
```

Comment:
Unless this is in a `@NonCPS` annotated method, this type of iteration throws an exception in our Jenkins version. You can use traditional `for ( int i = 0.... )` style or versions = some_new_noncps_method( packages ).